### ISSUE TITLE
How to run the docker?

### ISSUE THREAD

**Issue by Oleg26Dev** [localstack/localstack#5158]
### Is there an existing issue for this?

- [X] I have searched the existing issues and read the documentation

### Question

I run the docker in Windows 10
>docker run -it -e DEBUG=1 localstack/localstack

it is starting some time and finally I see the log (attached). Is this normal so many errors in the log? 
Or may be I do something wrong? 
[LocalStackLog.txt](https://github.com/localstack/localstack/files/7755279/LocalStackLog.txt)

**Comment by dfangl** [comment]
The error messages are sadly currently there, if your run with `DEBUG=1`.
Though you can run LocalStack like that, you will not be able to communicate with it like this.
Please look at https://docs.localstack.cloud/get-started/ for different ways to start LocalStack!

**Comment by Oleg26Dev** [comment]
Thank you.
I run 
>docker run --rm -it localstack/localstack

no error messages - OK.

Is there any Web UI ? i.e. how can I check how DynamoDB works and how cab I create a table on DynamoDB?

**Comment by dfangl** [comment]
Hi! First of all, you need to start localstack in a manner https://docs.localstack.cloud/get-started/ describes. If you want to start it manually with docker, please run (at least):
```
docker run --rm -it -p 4566:4566 -p 4571:4571 localstack/localstack
``` for the ports to be open.
Then you can communicate with LocalStack either via command line interface using `awslocal` or `aws` (https://docs.localstack.cloud/integrations/aws-cli/) or using the web interface on https://app.localstack.cloud (requires signup).

**Comment by dfangl** [comment]
I will close this issue now as the primary issue has been resolved (same as #5177).
If you have any more questions, please do not hesitate to open another issue, thanks!
